### ISSUE TITLE
UUID to name batch request

### ISSUE THREAD

**Issue by Flaflo** [Cydhra/Nidhogg#3]
there is 
```kotlin
fun getUUIDsByNames(...names)
```
a nice addition to that would be
```kotlin
fun getNamesByUUIDs(...uuids)
```

**Comment by Cydhra** [comment]
Are you suggesting a method, that spawns a thread and retrieves the names abiding by the rate limits?

**Comment by Flaflo** [comment]
no, mojang provides a way to request up to 100 uuids by names

**Comment by Cydhra** [comment]
yes. But it doesn't provide a way to request up to 100 names by uuids, so how exactly should `getNamesByUUIDs` be implemented?

**Comment by Flaflo** [comment]
well thats a problem, maybe find a solution or workaround without fucking with the rate limits

**Comment by Cydhra** [comment]
well I can provide such higher-level wrappers, I just wanted to know, if thats the thing you wanted.

**Comment by Flaflo** [comment]
that would be great

**Comment by Cydhra** [comment]
I implemented batch requests. There is no (known) rate limit for requesting different profiles, so it was very easy. I did not take any measures to prevent you from requesting the same profile too often (there is a rate limit to that), but I think there is no reason why you would do that anyway, so its your responsibility.

See the [unit test](https://github.com/Cydhra/Nidhogg/blob/d0682d81d4b3c9208640f3591246270a3c4052c8/src/jvmTest/kotlin/net/cydhra/nidhogg/BatchRequestsTest.kt) for an example on how to use the API.
